alvr/session/src/settings.rs

```diff
@@ -1548,10 +1548,8 @@ pub struct Patches {
 }
 
 #[derive(SettingsSchema, Serialize, Deserialize, Clone)]
-pub enum NewVersionPopupConfig {
-    Show,
-    HideWhileVersion(String),
-    AlwaysHide,
+pub struct NewVersionPopupConfig {
+    pub hide_while_version: String,
 }
```

Comment:
This isn't exactly great instead of just directly putting the String there, but ig it's fine? Idk how else to nicely document that that's the meaning